### ISSUE TITLE
Store profiler: business features step improvements

### ISSUE THREAD

**Issue by pmcpinto** [woocommerce/woocommerce-admin#6381]
**Description:** the experiment implemented in #5287 showed positive results. Recap here: pbIJXs-GS-p2. We're going to make some improvements in the flow and start targeting more people. I'll add the rest of the acceptance criteria early next week.

**Acceptance Criteria ( In order of Priority ):**
- [ ] Remove the flow created in #4579
- [ ] Remove the industry restriction and display this flow if the user selected one of the following countries in the store details: US | BR | FR | ID | GB | DE | VN | CA | PL | MY | AU | NG | GR | BE | PT | DK | SE | JP
- [ ] Include WooCommerce Payments if the user selected one of the following countries: US, AU, CA, GB, IE and NZ
- [ ] If the user installs Jetpack and/or WooCommerce Shipping & Tax, he should be redirected to the Jetpack sign up/log in flow at the end of the profiler. As we have now with the original flow
- [ ] Include Mailpoet in the list for any country within the "Grow your store" section: 
Copy: Level up your email marketing with [Mailpoet](https://www.mailpoet.com/woocommerce-email-marketing/)
wp.org link to download plugin: https://wordpress.org/plugins/mailpoet/
- [ ] Display extensions in the following order:
Mailpoet
Facebook
Google Ads
Mailchimp
Creative Mail
- [ ] Change the Creative Mail copy to: Emails made easy with Creative Mail
- [ ] Exclude the WooCommerce Shipping mention if the user is not in the US
- [ ] Include WooCommerce Tax if the user is in one of the following countries: US | FR | GB | DE | CA | PL | AU | GR | BE | PT | DK | SE
- [ ] The WooCommerce Tax copy should be the following: Get automated sales tax with [WooCommerce Tax](https://woocommerce.com/products/tax/)

**Event tracking:**
Currently we have one event that tracks the activity in the two tabs: `wcadmin_storeprofiler_store_business_features_continue` with just one event we're not able to understand if users are dropping in the business details or free features tab. It would be better to have two separate events:
- We can keep `wcadmin_storeprofiler_store_business_features_continue` to track the activity in the free features tab and keep all the associated event props: `all_extensions_installed`, `install_wcpay`, etc
- Add a new one just for the business details tab. We can call it `wcadmin_storeprofiler_store_business_details_continue_variant`. And this one should include all the props associated with this step: `used_platform`, `revenue`, `product_number`, `already_selling`, `used_platform_name`
Event name: `wcadmin_storeprofiler_store_business_features_link_click`
Event description: When the user clicks on one of the extensions link
Event prop: `extension_name` (value: name of the extension that was clicked)

**Additional links:**
Recap of the initial experiment: pbIJXs-GS-p2.

**Comment by pmcpinto** [comment]
@elizaan36 can you review the AC?

Regarding WC Shipping & Tax, we're only talking about printing shipping labels. But there's a chance that some users don't want shipping labels but need automated taxes (eg: digital products) and they don't install WC Shipping and Tax because we don't talk about automated taxes. I think the best option is to also present WooCommerce Tax as a separate option. Here's a rough mockup: https://d.pr/i/qeobMS
WDYT?

**Comment by timmyc** [comment]
> Include WooCommerce Payments if the user selected one of the following countries: US, AU, CA, GB, IE and NZ

Per the conversation in paJDYF-15E-p2, it seems like we should put this behind a feature flag for now so it can be turned off by default. Or a more lightweight solution might be to have it filterable.

> Include Mailpoet in the list for any country: wp.org link to download plugin | copy and placement in the list TBD

So we are okay with having 3 email extensions possibly installed by default?

Overall the criteria seems quite detailed. I do feel that this needs to be created into an epic, broken down into child issues that are prioritized in order of importance. 3 week sprint cycles go by quick so we might need to prioritize the work here to ensure we hit the key items in our 2.2 release.

Wanting to pull in @samueljseay since he has fresh experience in this space.

**Comment by elizaan36** [comment]
Regarding WCShip and WCTax:
Your mockup looks good @pmcpinto ! I'd just get "Get" to keep the format of starting with a verb. So the copy would be "Get automated taxes with _WooCommerce Tax_".
Another nice-to-have would be to hide "Print shipping labels..." for users who've selected digital orders only.

In terms of the three email extensions, definitely not ideal even if we leave one unchecked by default. I just read through your notes here Pedro (pbIJXs-G8#comment-2155) and sounds like there's some appetite for moving 2 of the email extensions to the task list. Is that something we can do in parallel to adding MailPoet to the wizard?

**Comment by pmcpinto** [comment]
>  I just read through your notes here Pedro (pbIJXs-G8#comment-2155) and sounds like there's some appetite for moving 2 of the email extensions to the task list. Is that something we can do in parallel to adding MailPoet to the wizard?

I think we don't have the bandwidth to implement a new task in the upcoming weeks.

**Comment by pmcpinto** [comment]
> I do feel that this needs to be created into an epic, broken down into child issues that are prioritized in order of importance. 3 week sprint cycles go by quick so we might need to prioritize the work here to ensure we hit the key items in our 2.2 release.

@timmyc I already prioritized the acceptance criteria and I'm happy to break it into child issues if necessary.

**Comment by timmyc** [comment]
Thanks for ordering items in level of priority @pmcpinto, yes I do still think we should break these into child issues and convert this to an epic.

**Comment by pmcpinto** [comment]
> Thanks for ordering items in level of priority @pmcpinto, yes I do still think we should break these into child issues and convert this to an epic.

@timmyc I was not able to convert this issue into an epic, so I ended up creating a new epic plus the child issues: #6485
I'm going to close this one.
